docs/chapters/abc/content.en-GB.md

```diff
@@ -47,7 +47,7 @@ And
   u(t)_{cubic} = \frac{(1-t)^3}{t^3 + (1-t)^3}
 \]
 
-So, if we know the start and end coordinates, and we know the *t* value, we know C, without having to calculate the `A` or even `B` coordinates. In fact, we can do the same for the ratio function: as another function of `t`, we technically don't need to know what `A` or `B` or `C` are, we can express it was a pure function of `t`, too.
+So, if we know the start and end coordinates and the *t* value, we know C without having to calculate the `A` or even `B` coordinates. In fact, we can do the same for the ratio function. As another function of `t`, we technically don't need to know what `A` or `B` or `C` are. We can express it too as a pure function of `t`.
```

Comment:
If you want to include a "too", then it'll have to become "It, too, can be expressed as a pure function of `t`."

Reply:
Much better.